packages/components/switch/src/switch.tsx

```diff
@@ -123,26 +123,28 @@ export const Switch = forwardRef<SwitchProps, "input">((props, ref) => {
       />
 
       {cloneIcon ?? (
-        <ui.span
-          className={cx("ui-switch__track", className)}
-          __css={{
-            alignItems: "center",
-            boxSizing: "content-box",
-            display: "inline-flex",
-            flexShrink: 0,
-            justifyContent: "flex-start",
-            ...styles.track,
-          }}
-          {...getIconProps()}
-        >
-          <motion.span
-            className={cx("ui-switch__thumb", className)}
-            data-checked={dataAttr(isChecked)}
-            layout
-            transition={transition}
-            __css={{ ...styles.thumb }}
-          />
-        </ui.span>
+        <motion.div layout layoutRoot>
```

Comment:
@108yen 

Why does `motion.div` `layout` and `layoutRoot` solve this problem?

Reply:
@hirotomoyamada 
Check this.
https://www.framer.com/motion/layout-animations/#troubleshooting

Reply:
Why don't you set `layout` and `layoutRoot` on the `span` element?

Reply:
@hirotomoyamada 

The `getIconProps()` result type is different from the `motion.span` prop type.